src/rp2_common/pico_btstack/CMakeLists.txt

```diff
@@ -232,21 +232,27 @@ if (EXISTS ${PICO_BTSTACK_PATH}/${BTSTACK_TEST_PATH})
 
     # Make a GATT header file from a BTstack GATT file
     # Pass the target library name library type and path to the GATT input file
+    # To add additional directories to the gatt #import path, one "[custom_path]"
+    # argument for each custom_path to the end of the argument list.
```

Comment:
Perhaps this description could be simplified to something like:
"To add additional directories to the gatt #import path, add them to the end of the argument list."
?

Reply:
done